app/dashboard/templates/profiles/profile.html

```diff
@@ -4,11 +4,11 @@
   <head>
     {% include 'shared/head.html' %}
     {% include 'shared/cards_pic.html' %}
-    <link rel="stylesheet" href={% static "v2/css/dashboard.css" %}>
-    <link rel="stylesheet" href={% static "v2/css/profile.css" %}>
-    <link rel="stylesheet" href={% static "v2/css/tag.css" %}>
+    <link rel="stylesheet" href={% static "v2/css/dashboard.css" %} />
+    <link rel="stylesheet" href={% static "v2/css/profile.css" %} />
+    <link rel="stylesheet" href={% static "v2/css/tag.css" %} />
     <link rel="stylesheet" href="{% static "v2/css/kudos/styles.css" %}" />
-    <link rel="stylesheet" href={% static "v2/css/tabs.css" %}>
+    <link rel="stylesheet" href={% static "v2/css/tabs.css" %} />
```

Comment:
No syntactical reasoning for adding closures to `link`

Reply:
Same here for "void elements". I always thought every element needs closure, but apparently, not... My bad, sorry.